src/compiler/reviews.md

```diff
@@ -127,217 +104,162 @@ bring PRs into good shape and meet the above criteria:
-  willing or able to accept the large change as-is. In such cases, the team
-  should make a decision to postpone or close, and clearly communicate the
-  decision to the PR author to explain the reasoning. It is very frustrating if
-  a PR stalls for many months only for it to be rejected anyway.
-
-
-[MCP]: https://forge.rust-lang.org/compiler/mcp.html
-[whats-a-major-change]:
-    https://forge.rust-lang.org/compiler/mcp.html#what-constitutes-a-major-change
-
+Reviewers are **not** expected to stomach PRs that are very large and complicated. It is expected
+from contributors to split their work to make a review feasable, for example a series of more
+digestible PRs which are individually more logically self-contained. In general, before submitting
+large impact changes, it is expected the contributor to have discussed the design previously with
+the relevant team(s) so it is contributor's duty to reference such discussions.
+
+When in doubt, bring the PR to the attention of the team (through zulip threads and/or nominate
+for compiler triage meeting), and the team can decide if:
+
+- The team can find suitable reviewers who can aid the PR author to break up the large change into
+  smaller logical PRs that are possible to review on their own, but also in the context of the
+  larger change.
+- The team does not have the bandwidth, or team members are is not ready or willing or able to
+  accept the large change as-is. In such cases, the team should make a decision to postpone or
+  close, and clearly communicate the decision to the PR author to explain the reasoning. It is
+  very frustrating if a PR stalls for many months only for it to be rejected anyway.
 
 ## Technical Aspects of Reviewing
+Every PR that lands in the compiler and its associated crates must be reviewed by at least one
+person who is knowledgeable with the code in question.
 
-Every PR that lands in the compiler and its associated crates must be reviewed
-by at least one person who is knowledgeable with the code in question.
+When a PR is opened, you can request a reviewer by including `r? @username` in the PR description.
+If you don't do so, rustbot will automatically assign someone from the pool of reviewer candidates,
+determined by the files affected.
 
-When a PR is opened, you can request a reviewer by including `r? @username` in
-the PR description. If you don't do so, rustbot will automatically assign
-someone from the pool of reviewer candidates, determined by the files affected.
+It is common to leave a `r? @username` comment at some later point to request review from someone
+else. This will also reassign the PR.
 
-It is common to leave a `r? @username` comment at some later point to request
-review from someone else. This will also reassign the PR.
-
-It is possible to request reviews from multiple reviewers, for example
+It is possible to request reviews from multiple reviewers, for example:
```

Comment:
Suggestion: probably remove the "request reviews from multiple reviewers" bit here, I think triagebot doesn't really handle this use case very well.

Reply:
Fixed!